operations/mimir/consul.libsonnet

```diff
@@ -2,11 +2,12 @@ local consul = import 'consul/consul.libsonnet';
 
 {
   _config+:: {
-    consul_replicas: 1,
+    consul_enabled: true,
```

Comment:
We should mention this new config option in changelog, so that users can learn about it.

Reply:
Added a CHANGELOG entry.